sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,14 +1,8 @@
 # Release History
 
-## 1.2.0-beta.2 (Unreleased)
+## 1.2.0 (Unreleased)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+No changes since `1.2.0-beta.1`.
```

Comment:
I think we need one of the standard headers for the changelog integration to work.

@weshaggard  can you confirm if we need to add at least one of the [Features Added, Breaking changes, Bug Fixed, etc] or if the eng scripts would work anyway?

cc @danieljurek

Reply:
It looks like you need at least one "Recommended section" https://github.com/Azure/azure-sdk-tools/blob/7f3b9914c1e8466cecad467b0f6ec4e8cd1b2598/eng/common/scripts/ChangeLog-Operations.ps1#L220 

```
("Features Added", "Breaking Changes", "Bugs Fixed", "Other Changes")
```
I would probably put this in an `Other Changes` section and I think you'll pass validation.

Reply:
Correction... having no recommended sections results only in a warning, not a failure. I think this will work as written.